src/ResourceManagement/HDInsight/HDInsight/Generated/Models/SecurityProfile.cs

```diff
@@ -43,7 +43,7 @@ public ActiveDirectoryConfiguration ActiveDirectoryConfiguration
         private string _roleAdminGroupDN;
         
         /// <summary>
-        /// Optional. Gets or sets the Distinguished Name for role
+        /// Optional. Optional. Gets or sets the Distinguished Name for role
```

Comment:
@iyerrb can you remove this extra "Optional" from the two properties that have it in their summary in this file?